.github/workflows/go.yml

```diff
@@ -124,3 +124,27 @@ jobs:
           npm test --prefix unsorted/sysl_js
         env:
           NPM_AUTH_TOKEN: "SOME-RANDOM-KEY"
+
+  build-docker:
+    name: Builds the sysl docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@master
+
+      - name: Build and push docker image
```

Comment:
Why push docker image in go-test workflow?

Reply:
This only builds it in the test workflow, to ensure that the docker image is still buildable.

If you look at like 149, push is set to false, so the image won't be published, only built